### ISSUE TITLE
Bandwidth filter. Multiple issues

### ISSUE THREAD

**Issue by HansonElectronics** [smeighan/xLights#1591]
The first issue is that if you select the "custom filtered waveform" once you need to change to 1 of the other waveform choices before the "midi not range" box appears to allow you to change the filter settings.

The second issue is that with the filter turned on there are peculiar waveforms when on custom or alto waveforms and zoomed in. This is particularly evident at higher frequencies. Can be seen clearly on attached sequence and zoomed in (not sure if it's specific to 25fps sequence setting). It's like an aliasing issue.

Third issue is that the filters look like they are maybe 3db per octave filters which provide little attenuation outside of their passbands. For a standard singing track the visible difference between the raw and filtered waveforms is small. The attached sweep sequence shows the filter differences. With Audacity it's possible to do a sharp cutoff of 60dB over a few/few hundred Hz.

[Sweep.zip](https://github.com/smeighan/xLights/files/3345273/Sweep.zip)

**Comment by keithsw1111** [comment]
#1 is what it is ... I cant change the behaviour of the menus.

Then i think i will just remove it unless you can come up with a better algorithm ... pretty much everything i found was awful. Either poorly coded or attrociously documented with the units not even specified.

**Comment by HansonElectronics** [comment]
I discovered how I was getting the weird waveforms. I loaded a fresh audio file via the sequence settings. A save and re-open of the xlights gets rid of the effect. Zooming in to maximum zoom at the righthand end of the sequence shows it up when on the alto or custom filter (8hz-3520hz). If changing to the audio file with that custom filter then the same stop/start visual effect is through the entire sequence except first 0.2s.
![Weird effect](https://user-images.githubusercontent.com/50623242/60439160-a374ba00-9c55-11e9-82f7-aa15974d7ed2.jpg)
[weird waveform.zip](https://github.com/smeighan/xLights/files/3345769/weird.waveform.zip)
